python/arcticdb/version_store/library.py

```diff
@@ -907,6 +907,56 @@ def read_metadata(self, symbol: str, as_of: Optional[AsOf] = None) -> VersionedI
         """
         return self._nvs.read_metadata(symbol, as_of)
 
+    def read_metadata_batch(
+        self, symbols: List[Union[str, ReadRequest]], query_builder: Optional[QueryBuilder] = None
+    ) -> List[VersionedItem]:
+        """
+        Reads multiple symbols.
+
+        Parameters
+        ----------
+        symbols : List[Union[str, ReadRequest]
+            List of symbols to read.
+            Params columns, date_range and query_builder from ReadRequest are not used
+        
+        query_builder: This parameter is not used
```

Comment:
Why include it then?

Reply:
And similar for the `ReadRequest` - it is not the right type to use here (the giveaway is that most of its attributes are not suitable for this method).

Reply:
@poodlewars we discussed. We decided that whilst not perfect, this is preferable to creating a `ReadMetadataRequest` or having a seperate `as_of`s parameter.